test/Test/Toml/Gen.hs

```diff
@@ -187,6 +190,36 @@ genToml = Gen.recursive
         arr <- Gen.list (Range.linear 1 10) genToml
         pure (key, NE.fromList arr)
 
+genTValue :: Gen TValue
+genTValue = Gen.element [ TBool
+                        , TInteger
+                        , TDouble
+                        , TText
+                        , TZoned
+                        , TLocal
+                        , TDay
+                        , THours
+                        , TArray
+                        ]
```

Comment:
I guess we can use [`enumBounded`](https://hackage.haskell.org/package/hedgehog-1.0.3/docs/Hedgehog-Internal-Gen.html#v:enumBounded) here, if we derive `Enum` for `TValue` 🙂